modules/imgproc/src/drawing.cpp

```diff
@@ -1159,7 +1159,7 @@ FillConvexPoly( Mat& img, const Point2l* v, int npts, const void* color, int lin
     edge[0].x = edge[1].x = -XY_ONE;
     edge[0].dx = edge[1].dx = 0;
 
-    ptr += img.step*y;
+    ptr += (int64_t)img.step*y;
```

Comment:
`int64_t` is C++11 stuff and it is not available on 3.4 branch.
3.4 branch must use `cv::int64` instead.